WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1775,8 +1776,16 @@ private void updateAndSavePostAsyncOnEditorExit(@NonNull final OnPostUpdatedFrom
 
     private UpdateFromEditor updateFromEditor(String oldContent) {
         try {
-            String title = (String) mEditorFragment.getTitle();
-            String content = (String) mEditorFragment.getContent(oldContent);
+            String title, content;
+            // To reduce redundant bridge events emitted to the Gutenberg editor, we get title and content at once
+            if (mEditorFragment instanceof GutenbergEditorFragment) {
```

Comment:
❓ I'm wondering if we need this `if` statement. Both Guteneberg fragment and Aztec fragment provide `getTitleAndContent(..)` method, right?

Reply:
Good point! I refactored the implementation to remove `getTitle` method from the interface (since it is no longer used anywhere), and now that method is private in Aztec. This allows us to simplify this and remove the conditional here.